src/item-extender/item-hooks-provider.ts

```diff
@@ -10,6 +10,9 @@ class CustomItemHooksProvider implements ItemHooksProvider {
         } else {
             // tslint:disable-next-line:no-console
             console.log(`A new item is being created`);
+
+            // tslint:disable-next-line:no-console
+            console.log();
```

Comment:
why we have two consecutive console lgos ?

Reply:
removed